web/src/routes/OrganizationsList/Organization/Tabs/Settings/GeneralSettings.tsx

```diff
@@ -0,0 +1,341 @@
+import {useEffect, useState, ReactNode} from 'react';
+import {
+  Flex,
+  FormGroup,
+  Form,
+  FormAlert,
+  TextInput,
+  FormSelect,
+  FormSelectOption,
+  ActionGroup,
+  Button,
+  Alert,
+  AlertGroup,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import {useCurrentUser} from 'src/hooks/UseCurrentUser';
+import {useOrganization} from 'src/hooks/UseOrganization';
+import {useOrganizationSettings} from 'src/hooks/UseOrganizationSettings';
+import {IOrganization} from 'src/resources/OrganizationResource';
+import {humanizeTimeForExpiry, getSeconds, isValidEmail} from 'src/libs/utils';
+import {addDisplayError} from 'src/resources/ErrorHandling';
+import {useQuayConfig} from 'src/hooks/UseQuayConfig';
+import {useUpdateUser} from 'src/hooks/UseCurrentUser';
+
+type validate = 'success' | 'warning' | 'error' | 'default';
+const timeMachineOptions = {
```

Comment:
Custom tag options set by `TAG_EXPIRATION_OPTIONS` doesn't seem to be supported. For example if I set `TAG_EXPIRATION_OPTIONS` to `2d, 5d, 6w` nothing get's rendered in the list